applications/PoromechanicsApplication/python_scripts/poromechanics_fracture_U_Pw_solver.py

```diff
@@ -0,0 +1,442 @@
+from __future__ import print_function, absolute_import, division # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+#import kratos core and applications
+import KratosMultiphysics
+import KratosMultiphysics.SolidMechanicsApplication
+import KratosMultiphysics.PoromechanicsApplication as KratosPoro
+import KratosMultiphysics.FluidDynamicsApplication as KratosCFD
+
+# Check that KratosMultiphysics was imported in the main script
+KratosMultiphysics.CheckForPreviousImport()
+
+
+def CreateSolver(main_model_part, custom_settings):
+
+    return FractureUPwSolver(main_model_part, custom_settings)
+
+
+class FractureUPwSolver(object):
```

Comment:
this does not derive from PythonSolver ?

Reply:
Ah I see it is kept for retro-compatibility

Reply:
Yes, this is not up to current standards but in order to to adapt it, a complete modification in the fracture propagation utility would be necessary and I don't have the time to do so.
These changes are the minimal so that it still works...